patterns/base/select/select.html

```diff
@@ -0,0 +1,5 @@
+<select name="demo-select" id="demo-select" class="base--select">
```

Comment:
kind of nitpicky so feel free to ignore, but i'd change the name and id to something less closely resembling the class name (such as just "demo" or "select-demo")